server/pkg/pr12er/database.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package pkg.pr12er;
+
+import "google/protobuf/timestamp.proto";
+import "pkg/pr12er/messages.proto";
+
+option go_package = "github.com/codingpot/server/pkg/pr12er";
+
+message MappingTable {
+  repeated MappingTableRow rows = 1;
+}
+
+message MappingTableRow {
+  int32 pr_id = 1;
+  repeated string paper_arxiv_id = 2;
+  string youtube_video_id = 3;
+//  repeated Override overrides = 4;
```

Comment:
```suggestion
```

Reply:
Override is not defined yet, so I have commented it out